upgrading/manual_upgrades.adoc

```diff
@@ -507,80 +507,38 @@ updated.
 [[updating-policy-definitions]]
 == Updating Policy Definitions
 
-After a cluster upgrade, the default roles
+During a cluster upgrade, the default roles
 xref:../architecture/additional_concepts/authorization.adoc#roles[default
-cluster roles] are automatically updated. To check if all defaults are set as
-recommended for your environment, run:
-
-----
-# oc adm policy reconcile-cluster-roles
-----
+cluster roles] are automatically updated. 
 
 [WARNING]
 ====
-If you have customized default cluster roles and want to ensure a role reconciliation
-does not modify those customized roles, annotate them with `openshift.io/reconcile-protect`
-set to `true` when using the old Openshift policy format. When using the new RBAC
-roles, use `rbac.authorization.kubernetes.io/autoupdate` set to `false` instead.
-In doing so, you are responsible for manually updating those roles with any new
-or required permissions during upgrades.
+If you customized default cluster roles and want to ensure a role reconciliation
+does not modify those customized roles, set the
+`rbac.authorization.kubernetes.io/autoupdate` parameter to `false`.
```

Comment:
This needs a clear `oc` example on how to annotate RBAC (i.e. `clusterrole.rbac` vs `clusterrole`).

Reply:
ex: `oc annotate clusterrole.rbac system:webhook --overwrite rbac.authorization.kubernetes.io/autoupdate=false`